src/protocolgame.cpp

```diff
@@ -2841,6 +2849,9 @@ void ProtocolGame::AddPlayerStats(NetworkMessage& msg)
 	msg.add<uint16_t>(condition ? condition->getTicks() / 1000 : 0x00);
 
 	msg.add<uint16_t>(player->getOfflineTrainingTime() / 60 / 1000);
+
+	msg.add<uint16_t>(0); // xp boost time (seconds)
+	msg.addByte(0); // ?
```

Comment:
This byte indicates whether the player may buy further experience boosts in the store.
It will enable/disable the "Exp Boost" button in the skill window.